fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/FDBRecordStoreBase.java

```diff
@@ -1561,15 +1561,15 @@ default int getPipelineSize(@Nonnull PipelineOperation pipelineOperation) {
          * @return context the record context / transaction to use
          */
         @Nullable
-        FDBRecordContext getContext();
+        FDBRecordContext getRecordContext();
```

Comment:
I think we want to deprecate the old method rather than rename it. Somewhat awkwardly, I think the "right" way to do this for an interface like this is deprecate this method but leave it as the thing that sub-interfaces need to implement, then the *new* method suppresses deprecation warnings and calls the old one.

That's what is needed if we don't want to break implementors. If we only want to avoid breaking callers, then it's sufficient to make the new one and make the deprecated one call the new one.